cogs/yourlsClient/core.py

```diff
@@ -0,0 +1,55 @@
+import asyncio
+import logging
+import os
+
+import discord
+
+from redbot.core import Config, data_manager
+from redbot.core.bot import Red
+
+from .api import YOURLSClient
+from .constants import BASE_GUILD, KEY_API, KEY_SIGNATURE
+from .exceptions import YOURLSNotConfigured
+
+
+class Core:
+    def __init__(self, bot: Red):
+        super().__init__()
+        self.bot = bot
+        self.config = Config.get_conf(self, identifier=5842647, force_registration=True)
+        self.config.register_guild(**BASE_GUILD)
+        self.loop = asyncio.get_running_loop()
+
+        saveFolder = data_manager.cog_data_path(cog_instance=self)
+        self.logger = logging.getLogger("red.luicogs.YOURLS")
+        if not self.logger.handlers:
+            logPath = os.path.join(saveFolder, "info.log")
+            handler = logging.FileHandler(filename=logPath, encoding="utf-8", mode="a")
+            handler.setFormatter(
+                logging.Formatter("%(asctime)s %(message)s", datefmt="[%d/%m/%Y %H:%M:%S]")
```

Comment:
Unrelated to the changes in this PR, but could you also fix this to be YYYY-MM-DD?

Reply:
Yes.